evelyn_chen/alpha_challenges/conditionals.md

```diff
@@ -27,13 +43,34 @@ Create a Ruby method called `temperature_status` that takes three arguments: the
   - If the air conditioner is non-functional and the current temperature is below the the desired temperature, print the string "Fix the A/C whenever you have the chance. It's cool."
 
 ```ruby
-# write your code here
+def temperature_status(current_temp, AC_functional, desired_temp)
+  if AC_functional and current_temp >= desired_temp
+    'Turn on the A/C, please.'
+  elsif current_temp >= desired_temp
+    'Fix the A/C now! It is hot!'
+  else current_temp <= desired_temp
+    'Fix the A/C whenever you have the chance. It is cool.'
+  end
+end
+#I tried using unless in code below but I'm unsure if it works. Would this work?
```

Comment:
Unless is the opposite of `if`, meaning it's the same as `if not` or `if !`. This means you don't normally use it with an if statement. 

With "if":
```
if my_number == 2
    'my number is 2'
else
    'my number is not 2'
```

With unless:
```
unless my_number == 2
    'my number is not 2'
else
    'my number is 2'
```

In the case of this question, for the 2nd line you could use:
```print 'Fix the A/C now! It is hot!' unless AC_functional && current_temp < desired temp```

Reply:
Oh, I see, got it! Thank you so much!

Reply:
Then if I include unless, how would I change the first conditional:
`print 'Turn on the A/C, please.' if AC_functional and current_temp >= desired_temp`?

Reply:
In this case we could use: 
```print 'Turn on the A/C, please.' unless !AC_functional and current_temp < desired_temp.```

But in this question, because there are so many different cases, I think the unless is unnecessary. I personally find unless useful when you're checking for a single exception. With lots of potential scenarios, I find the if statement is a lot easier to read and understand :)

Reply:
ok! So `if not` produces the same effect right?

Reply:
yep. I would suggest using the `!` to negate the if. like this:
```
print 'Turn on the A/C, please.' if AC_functional and current_temp > desired_temp
print 'Fix the A/C now! It is hot!' if current_temp > desired_temp and !AC_functional
print 'Fix the A/C whenever you have the chance. It is cool.' if current_temp < desired_temp and !AC_functional
```

Reply:
Thank you!